src/gui/main.cpp

```diff
@@ -65,6 +67,9 @@ CommandLineParseResults parseCommandLine(QCommandLineParser &parser, QString *er
     const QCommandLineOption backgroundOption(QStringList() << "b" << "background",
                                               QObject::tr("Starts in background, without displaying the main window."));
     parser.addOption(backgroundOption);
+    const QCommandLineOption daemonOption(QStringList() << "x" << "daemon",
+                                              QObject::tr("Starts as daemon without display connection."));
```

Comment:
Please make it obvious that this use case is not officially supported. Fixes will always be accepted of course.

Reply:
What is your resistance to supporting this use case? I often run without X on my systems and want my keyboard to reflect my profiles at login. It's especially useful where I'm connected to a KVM switch and color-code the keys according to which system is selected on the KVM.